zproject/backends.py

```diff
@@ -788,6 +788,7 @@ def get_or_build_user(self, username: str,
         """This is used only in non-authentication contexts such as:
              ./manage.py sync_ldap_user_data
         """
+        logger = logging.getLogger("zulip.ldap")
```

Comment:
Isn't there already an `ldap_logger` variable for this?

Reply:
Ohh right, there is, changed to use that